README.md

```diff
@@ -1 +1,235 @@
+
+> Describe how a new developer could make use of your project.
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+- Github
+- Git
+
+<!--
+Example command:
+
+sh
+ gem install rails
+
+ -->
+
+### Setup
+
+Clone this repository to your desired folder:
+- cd my-folder
+- git clone git@github.com:myaccount/my-project.git
+<!--
+Example commands:
+
+sh
+  cd my-folder
+  git clone git@github.com:myaccount/my-project.git
+
+--->
+
+### Install
+
+Install this project with:
+- git clone 
+
+<!--
+Example command:
+
+sh
+  cd my-project
+  gem install
+
+--->
+
+### Usage
+
+To run the project, execute the following command:
+
+<!--
+Example command:
+
+sh
+  rails server
+
+--->
+
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="Ismail"></a>
+
+> Mention all of the collaborators of this project.
+
+👤 *Author1*
+
+- GitHub: [@githubhandle](https://github.com/ismailmunyentwari9)
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+>I will try to improve this project and add some new things in the future.
+
+- [ ] *[new_feature_1]*
+- [ ] *[new_feature_2]*
+- [ ] *[new_feature_3]*
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+> This is my first Microverse project, and if you like it you can make me happy by giving a star.
+
+If you like this project give a like.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+> Give credit to everyone who inspired your codebase.
+
+I would like to thank to Microverse for helping us to complete this project.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FAQ (optional) -->
+
+## ❓ FAQ <a name="faq"></a>
+
+> Add at least 2 questions new developers would ask when they decide to use your project.
+
+- *Can we clone it without Git?*
+
+  - No, you can can't clone without Git.
+
+- *Which langauges did you use to make this?*
+
+  - I used HTML and CSS
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE) licensed.
+
+NOTE: we recommend using the [MIT license](https://choosealicense.com/licenses/mit/) - you can set it up quickly by [using templates available on GitHub](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository). You can also use [any other license](https://choosealicense.com/licenses/) if you wish.
```

Comment:
- [ ] The link to your MIT is broken, which means that you don't have any file in your root directory named `./LICENSE`.

![image](https://user-images.githubusercontent.com/77106962/204624308-0ce0e808-2814-406e-a26e-9444da5a564b.png)

 To fix this you can create a license file by following the instructions in this [link](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository) or this [one](https://choosealicense.com/licenses/). Kindly fix it. 👍